src/Http/HttpResponse.php

```diff
@@ -61,8 +61,10 @@ public function getResponseBody()
      * @param int $expirySecondsFromNow Time the cookie should last for in seconds. Defaults to 2 weeks. Pass null to make this a session cookie.
      * @param string $path Web path the cookie should be available to - defaults to "/", the whole site
      * @param string $domain Domain the cookie should be available to - defaults to current subdomain. Set to ".domain.com" to make available to all subdomains.
+     * @param bool $secure Indicates that the cookie should only be transmitted via HTTPS - defaults to false
+     * @param bool $httpOnly Indicates that the cookie should only be transmitted via the HTTP Protocol - defaults to true
      */
-    public static function setCookie($name, $value, $expirySecondsFromNow = 1209600, $path = "/", $domain = null)
+    public static function setCookie($name, $value, $expirySecondsFromNow = 1209600, $path = "/", $domain = null, $secure = false, $httpOnly = true)
```

Comment:
the only issue here might be that we break places where the client does access the cookie. Could you ask @scottmcmurray  and @samnotsowise if they know of any places where this is the case?

Reply:
yeah, i've used cookies from js in a number of rhubarb projects i think.

Behavior changes like this should not be part of a minor version bump imo (and if we are trying to honour semver).